Formula/re-flex.rb

```diff
@@ -0,0 +1,22 @@
+class ReFlex < Formula
+  desc "The regex-centric, fast and flexible scanner generator for C++"
+  homepage "https://www.genivia.com/doc/reflex/html"
+  url "https://github.com/Genivia/RE-flex/archive/reflex-1.0.12.tar.gz"
+  sha256 "75a1761d9a6fd3fee6454e3b6ec954efbad76ece9b68633a07ada977af4cd300"
+
+  depends_on "boost"
+
+  def install
+    ENV["YACC"] = "#{Formula["bison"].bin}/bison -y"
+    # system "autoreconf", "-fi"
```

Comment:
This line can be removed if not needed.

Reply:
We needed `autoreconf -fi` when cloning from GithHub, due to timestamp changes that could break `./configure`. We don't need it for this formula. I'll remove it.